aviatrix/utils.go

```diff
@@ -20,6 +20,28 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
+func validateIPv6CIDR(i interface{}, k string) (warnings []string, errors []error) {
+	v, ok := i.(string)
+	if !ok {
+		errors = append(errors, fmt.Errorf("expected type of %q to be string", k))
+		return warnings, errors
+	}
+
+	ip, _, err := net.ParseCIDR(v)
```

Comment:
IIUC, this is meant to be a network CIDR, so we should validate that the ipmask result has a Size() of more than 0?

i.e like validation.IsCIDRNetwork does.